lib/msal-browser/src/interaction_client/SilentRefreshClient.ts

```diff
@@ -14,6 +14,10 @@ export class SilentRefreshClient extends StandardInteractionClient {
      * @param request
      */
     async acquireToken(request: CommonSilentFlowRequest): Promise<AuthenticationResult> {
+        if (request.silentTokenRetrievalStrategy === SilentTokenRetrievalStrategy.NetworkOnly) {
+            throw new ServerError(BrowserConstants.INVALID_GRANT_ERROR);
```

Comment:
Can you also add a log message here indicating that this if statement was entered?